src/Movie/Movie.js

```diff
@@ -0,0 +1,13 @@
+import React, { Component } from "react"
+import "./Movie.css"
+
+
+const Movie = (props) => {
+    console.log(props.movie)
+    return (
+        <div className="single-movie">
+            <img classname="single-movie-img"src={props.movie[0].backdrop_path} height="100px" />
```

Comment:
Be sure you are including an `alt` tag every time you display an image.  Try to make the alt tag specific and helpful info (like the specific movie's title or something).  Absolutely essential for screen readers.